docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+   a. Setup and manage your wallet accounts, currencies, balances etc.  
+   b. Obtain the wallet address or payment pointer for your digital wallet.
+
+### Test Data
+
+This table describes the different ways that you can set up a website that is monetized, and a website that isn’t monetized, so that you can test the Web Monetization extension.
+
+| Web monetized websites                                                                                                                                                                                                                      | Non monetized websites    |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------ |
+| One valid monetization link tag                                                                                                                                                                                                             | No monetization link tags |
+| Multiple valid monetization link tags with wallet addresses: <ul><li>from the same wallet providers</li><li>from different wallet providers</li><li>with different currencies</li><li>with some link tags enabled & some disabled</li></ul> | All disabled link tags    |
+
+### Test Playground
+
+Use the [Web Monetization playground](https://webmonetization.org/play/) to quickly and seamlessly create the conditions of a monetized website, to use only for testing. When you add wallet addresses or payment pointers to the playground, the playground becomes a monetized website that you can use the Web Monetization extension to pay
+
+On the playground, you can use real money from a real wallet, or you can use “play” money from a “test wallet”.
+
+**Important**
+
+1. The playground itself is not web monetized.
+2. The playground becomes monetized when you add one or more receiving wallet addresses to it.
+3. If your Web Monetization extension is connected to a real wallet, then your extension will facilitate payments from your real wallet (i.e. real money).
+4. Similarly, if your extension is connected to a play wallet (i.e. rafiki.money), then the extension facilitates payments using play money.
+5. If you add your own wallet address to the playground, and then use the extension to pay the playground, you are paying yourself.
+
+### Functional Test Cases
+
+These test cases are defined from the point of view of a website visitor that wants to use Web Monetization to pay.
+
+#### Risk Areas
+
+**Goal**: To lower the barriers to Web Monetization adoption.
+**Approach**: Identify and prioritize risks to the adoption of the WM extension.
+Based on these risks, the sections that follow will detail the actual test cases.
+
+<!-- prettier-ignore-start -->
+We use 4 risk priorities: `critical` | `high` | `medium` | `low`
```

Comment:
Four are listed, but only criticals and highs are listed. Two suggestions: First, list them in order of importance with criticals appearing first. Second, if everything really is only critical or high, remove mention of medium and low.

Reply:
Cool, and done, thanks